neqo-transport/src/connection.rs

```diff
@@ -1259,9 +1295,17 @@ impl Connection {
                 SentPacket::new(now, ack_eliciting, tokens, packet.len(), in_flight),
             );
 
+            if self.first_1_rtt_pn == None && epoch == 3 {
```

Comment:
I know that we aren't running clippy here in pedantic mode, but...

```suggestion
            if self.first_1_rtt_pn.is_none() && epoch == 3 {
```